lib/js/index.js

```diff
@@ -202,7 +209,7 @@ class DateTimePicker extends Events {
   }
 
   onChangeDate(dateString) {
-    const newValue = moment(this.value);
+    const newValue = moment(this.options.default);
```

Comment:
I did it because when you clicked on a different date it changes the language back to English. This is caused by using `this.value`, which is the default moment object that has the English locale only available. `this.options.default` is the received moment object and can containts, therefore, more languages.

However, as you said it doesnt look right. Looking deeper on the code I have found other cases where this unwelcome behaviour appear.